File: HW1/fourargs.py
import sys
if len(sys.argv)>4:
    for i in range(1,5):	
        sys.stdout.write (sys.argv[i]+'\n') 
    for i in range(5,len(sys.argv)):
        sys.stderr.write (sys.argv[i]+'\n') 
else:
    for i in range(1,len(sys.argv)):
        sys.stdout.write(sys.argv[i]+'\n')
# Copyright year ShuangZhao zs1995@bu.edu

File: HW1/fourargs.cpp
#include<iostream>
int main(int argumentscount, char **arguments){
	for(int i=1;i<=4;i++){
	    std::cout<<arguments[i]<<'\n';
    }
    for(int j=5;j<=argumentscount;j++){
    	std::cerr<<arguments[j]<<'\n';
    }
	return 0;
}
// Copyright year ShuangZhao zs1995@bu.edu